File: Cap 04/Topico 4.6/Topico 4.6.3/Operador de Negacao/nottest.pl
=pod

 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.6.3
 * 
 * Programa ..: OPERACAO DE NEGACAO ".NAO. / NOT"
 
=cut

use strict;
use warnings;

my ($A, $B, $X, $C);

print "TESTE LOGICO OPERADOR: NOT\n\n";

print "Entre valor inteiro <A>: ";
$A = <STDIN>;

print "Entre valor inteiro <B>: ";
$B = <STDIN>;

print "Entre valor inteiro <X>: ";
$X = <STDIN>;

if (not ($X > 5)) {
    $C = $A + $B;
} else {
    $C = $A - $B;
}

print "O valor de C = $C\n";

File: Cap 04/Topico 4.6/Topico 4.6.3/Operador de Negacao/nottest.py
'''
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.6.3
 *
 * Programa ..: OPERAÇÃO DE NEGAÇÃO ".NÃO. / NOT"
'''

print("TESTE LOGICO OPERADOR: NOT")
print()
A = int(input("Entre valor inteiro <A>: "))
B = int(input("Entre valor inteiro <B>: "))
X = int(input("Entre valor inteiro <X>: "))
if not (X > 5):
  C = A + B
else:
  C = A - B
print("O valor de C = ", C)


